processing/src/main/java/io/druid/segment/DimensionSelector.java

```diff
@@ -149,6 +149,13 @@ default long getLong()
     return 0L;
```

Comment:
maybe need to add assertion, let's revisit https://github.com/druid-io/druid/issues/4888